ClassicLootManager/Modules/Rosters/RosterManager/IndirectSlotMap.lua

```diff
@@ -127,6 +127,22 @@ local IndirectMap = {
         [231882] = "INVTYPE_2HWEAPON",  -- Suppression Device Receipt
         [231995] = "INVTYPE_SHIELD",    -- Hardened Elementium Slag
         [231814] = "INVTYPE_2HWEAPON",  -- Chromatic Heart
+        --- SOD P6 ---
+        ---AQ40
+        [233369] = "INVTYPE_FEET",     -- Qiraji Bindings of Dominance
+        [233370] = "INVTYPE_FEET",     -- Qiraji Bindings of Command
+        [233371] = "INVTYPE_FEET",     -- Qiraji Bindings of Sovereignty
+        [20883] = "INVTYPE_HEAD",       -- Qiraji Glyphed Jewel
+        [20887] = "INVTYPE_HEAD",       -- Qiraji Engraved Jewel
+        --[235045] = "INVTYPE_2HWEAPON",  -- Imperial Qiraji Regalia --- Multiple different item types available from item, unsure what to set as
```

Comment:
Both items are in MH or 2H so 2H is the way to go.

Ilvl79 seems to be for both items